chia/wallet/cat_wallet/cat_wallet.py

```diff
@@ -192,7 +192,7 @@ async def create_wallet_for_cat(
                 assert isinstance(w, CATWallet)
                 if w.get_asset_id() == limitations_program_hash_hex:
                     self.log.warning("Not creating wallet for already existing CAT wallet")
-                    raise ValueError("Wallet already exists")
+                    return w
```

Comment:
Here's the main change

Reply:
Resolving, as this has been reviewed, and approved